src/Diagnostics.DataProviders/KustoClient.cs

```diff
@@ -64,11 +64,11 @@ public async Task<DataTable> ExecuteQueryAsync(string query, string cluster, str
         {
             var timeTakenStopWatch = new Stopwatch();
             var authorizationToken = await KustoTokenService.Instance.GetAuthorizationTokenAsync();
-            var kustoClientId = $"Diagnostics.{operationName ?? "Query"};{_requestId}##{0}";
+            var kustoClientId = $"Diagnostics.{operationName ?? "Query"};{_requestId}##{0}_{(new Guid()).ToString()}";
```

Comment:
Why is the string `"##0"` appended to `_requestId`? 
I wonder if this is intentional or a leftover `{0}` from a previous `string.Format()` operation.

Reply:
It most likely is, however I am a little reluctant to change it as we might be parsing this field somewhere and I don't want whatever (if anything ) is doing it.